src/checkbox-group/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import { checkboxGroup } from './middleware';
+import { Checkbox } from '../checkbox/index';
+import { RenderResult } from '@dojo/framework/core/interfaces';
+import theme from '@dojo/framework/core/middleware/theme';
+import * as css from '../theme/checkbox-group.m.css';
+
+type CheckboxOptions = { value: string; label?: string }[];
+
+interface CheckboxGroupProperties {
```

Comment:
In preparation for the examples and doc app, can you add the appropriate comments for each of the properties, like:

```
/** The name of the thing */
name: string;
```

Also does this not need to exported?